src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Consider providing a fallback font-family in case the main one doesn't load. This ensures that your text is always displayed in a font that closely matches the design intent.